### ISSUE TITLE
cheat satellite contracts with one vessel for two contracts

### ISSUE THREAD

**Issue by phwoelfel** [Arsonide/FinePrint#16]
I had two contracts to put an satellite into orbit around mun, one in a small about 10 degree inclined orbit and one in a higher more inclined orbit. I built a satellite, put it in the first orbit on arrival at the moon, waited for the contract to be complete and then changed my orbit to match the second orbit and this one also completed.
Maybe check if the vessel has already been used for a contract or require the satellite to be in a stable orbit for a certain time (like DMagic's contracts do)

**Comment by erendrake** [comment]
I dont think that is is necessarily a cheat, re-tasking a satellite is not a trivial task, in this case it worked but with a significant change in inclination it might not.

I think it would be pretty cool if a contract comes up and it turns out you could reassign a craft already in system and finish the contract. Having a duration for a stable orbit is a pretty neat idea too.
